### PR TITLE
Re-compute page every day by GitHub Action

### LINKED ISSUE #4: Rebuild the page every day using a cron task
So that the information is updated

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Re-compute page every day by GitHub Action. See reference https://jasonet.co/posts/scheduled-actions/
| Type?             | new feature
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/ps-monitor-module-releases/issues/4
| How to test?      | ~
| Possible impacts? | CI only

### CODE REVIEW

File: .github/workflows/build-and-deploy.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 name: Build and Deploy
 on:
+on:
```
Comment (ghost) [inline .github/workflows/build-and-deploy.yml]: I am not an expert in github action, but there are two times `on:` ?

Comment (kpodemski) [reply .github/workflows/build-and-deploy.yml]: I'm pretty sure it's just a typo :>

Comment (kpodemski) [inline .github/workflows/build-and-deploy.yml]:
```suggestion
```

Comment (matks) [reply .github/workflows/build-and-deploy.yml]: Yes 😅 

This proves that even if you are not an expert in a field you can always provide value by reviewing the contribution 👍

**Review by ghost — APPROVED** [review]

**Comment by matks** [comment]
Thank you for review

**Pull request merged**